include/ck/tensor_operation/gpu/device/impl/device_batchnorm_forward_impl.hpp

```diff
@@ -244,8 +246,8 @@ struct DeviceBatchNormFwdImpl : public DeviceBatchNormFwd<XDataType,
                     int testBlkGroupSize = (reduce_length_ + (K_BlockTileSize * iterations) - 1) /
                                            (K_BlockTileSize * iterations);
 
-                    // we want the blkGroupSize be not more than 128
-                    if(testBlkGroupSize <= 128)
+                    // we want the blkGroupSize be not more than 16
+                    if(testBlkGroupSize <= 16)
```

Comment:
why reduce the limit of testBlkGroupSize?

Reply:
`16` is enough to bring rich number of workgroups to occupy the CUs,  bigger BlkGroupSize leads to stronger competition when locking, not beneficial for performance